radis/lbl/calc.py

```diff
@@ -418,6 +420,15 @@ def _check_molecules_are_consistent(
             for mol in molecule_reference_set:
                 molecule_dict[mol][argument_name] = argument_dict[mol]
 
+    # Checking if `auto` mode
+    if optimization == "auto":
+        optimization = "min-RMS"
+        broadening_max_width = 300
+        if (wavenum_max - wavenum_min) < 300:
+            broadening_method = "fft"
+        else:
+            broadening_method = "voigt"
```

Comment:
So in case if the user gives all values that are `broadening_max_width` and `truncation` we should simple use the default truncation value and default `neighbour_lines`?
Also what will be the value of `neighbour_lines` based on different `spectral range`, `optimization` and `broadening_method`?